samples/AzureStorageQueuesPubSubSample/AzureStorageQueuesPubSub.cs

```diff
@@ -0,0 +1,141 @@
+using Azure.Storage.Queues;
+using Dapr.PluggableComponents.Components;
+using Dapr.PluggableComponents.Components.PubSub;
+
+internal sealed class AzureStorageQueuesPubSub : IPubSub
+{
+    private readonly ILogger<AzureStorageQueuesPubSub> logger;
+
+    private string? connectionString;
+    private string? queueName;
+    private TimeSpan pollInterval = TimeSpan.FromSeconds(5);
+    private int maxMessages = 5; 
+
+    public AzureStorageQueuesPubSub(ILogger<AzureStorageQueuesPubSub> logger)
+    {
+        this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+    }
+
+    #region IPubSub Members
+
+    public Task InitAsync(MetadataRequest request, CancellationToken cancellationToken = default)
+    {
+        this.connectionString = request.Properties["connectionString"];
+        this.queueName = request.Properties["queueName"];
+
+        if (request.Properties.TryGetValue("pollInterval", out var pollIntervalString))
```

Comment:
Should `pollInterval` be renamed to `pollIntervalSeconds` or something that clearly states what unit this expects? Or is this a grandfathered configuration mame?

Reply:
That's fair.  I've updated the property name as well as added a README for the sample to provide a little more context.